clio-server/src/test/scala/org/broadinstitute/clio/server/dataaccess/FailingPersistenceDAO.scala

```diff
@@ -9,7 +9,7 @@ import org.broadinstitute.clio.server.dataaccess.elasticsearch.{
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class FailingPersistenceDAO extends PersistenceDAO {
+class FailingPersistenceDAO extends PersistenceDAO(recoveryParallelism = 1) {
```

Comment:
any way to make this an optional and avoid having to provide it to get the no-op behavior?